### PR TITLE
Event format

### PR DESCRIPTION
**Pull request by h00dy**
- entity_name, check_name, tags added to event format
- initial vagrant config (requires further work)

### CODE REVIEW

File: vagrant/consul-launcher.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Install and run counsul agent
+CONSUL_DIR='/usr/bin'
+if [ ! -f $CONSUL_DIR/consul ]; then
+wget -t 2 -O /tmp/consul.zip https://dl.bintray.com/mitchellh/consul/0.5.2_linux_amd64.zip
```
Comment (deejay1) [inline vagrant/consul-launcher.sh]: consul 0.6 is out for some time now ;)

File: alamo_alerter/alerter.py [hunk 2]
```diff
@@ -61,6 +61,9 @@ def get_check_metric(self, check):
         # need plugin agnostic solution for this
         return check['fields'].get('url') or check['fields'].get('metric')
 
+    def merged_tags(self, check, trigger):
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: doc string?

**Comment by deejay1** [comment]
:+1:

**Comment by pnuckowski** [comment]
:+1:

**Pull request merged**